src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
```

Comment:
I would give the variable a different name, not a generic object, but an "order"

Reply:
Partly agree with you, but I use this method always with mapper object, so the whole picture seems nicely readable
orderResponceMapper.toDto()

Reply:
Strictly speaking every class instance is `object` in Java, so this naming can be not clear enough.
From my point of view it is better to use such names as `order`, `model`, or `entity`.
Please fix in other similar cases also.